java-checks/src/test/java/org/sonar/java/checks/tests/AssertionsCompletenessCheckTest.java

```diff
@@ -48,4 +53,21 @@ void test_non_compiling_because_of_removed_api() {
       .verifyIssues();
   }
 
+  @Test
+  void test_isSoftAssertionsExtensionClass(){
```

Comment:
I'm not super certain that it is the only way to test this case. You could not find a test case that would match?